### ISSUE TITLE
Can't download most URLs, including google.com

### ISSUE THREAD

**Issue by TZubiri** [scrapy/scrapy#5367]
### Description
Example code from tutorial (https://docs.scrapy.org/en/latest/intro/overview.html) can't download non-example URLS.

One example of a URL it fails to download is https://google.com

### Steps to Reproduce

1. Download example code from tutorial.
2. Remove the parse function.
3. Run and verify that the page was downloaded correctly.
4. Change the url to another server like google.
5. Run and verify that an error is encountered

**Actual behaviour:**

 ```

2022-01-20 18:09:07 [scrapy.core.scraper] ERROR: Error downloading <GET https://google.com>
Traceback (most recent call last):
  File "/home/tomi/.local/lib/python3.5/site-packages/scrapy/core/downloader/middleware.py", line 44, in process_request
    return (yield download_func(request=request, spider=spider))
twisted.web._newclient.RequestGenerationFailed: [<twisted.python.failure.Failure builtins.AttributeError: __exit__>]
```

**Reproduces how often:** Always

### Versions

```
Scrapy       : 2.3.0
lxml         : 3.8.0.0
libxml2      : 2.9.3
cssselect    : 1.1.0
parsel       : 1.6.0
w3lib        : 1.22.0
Twisted      : 20.3.0
Python       : 3.5.2 (default, Nov 12 2018, 13:43:14) - [GCC 5.4.0 20160609]
pyOpenSSL    : 20.0.1 (OpenSSL 1.1.1h  22 Sep 2020)
cryptography : 3.2.1
Platform     : Linux-4.15.0-142-generic-x86_64-with-Ubuntu-16.04-xenial
```

Thank you for your time,
Tomas.

**Comment by wRAR** [comment]
I can't reproduce this, of course.

**Comment by TZubiri** [comment]
Probably a pyOpenSSL error, as it was shared by 'requests'. Fixed by switching to a 3.6 installation.
